src/core/evaluator.js

```diff
@@ -978,7 +986,7 @@ class PartialEvaluator {
     });
   }
 
-  loadFont(fontName, font, resources) {
+  loadFont(fontName, font, resources, isButtonContext) {
```

Comment:
I'm not really a big fan of this annotation detail leaking through to the generic evaluator. Would it perhaps be slightly better to let the caller (the annotation code) provide a `fallbackFont` dictionary explicitly and pass that instead of this boolean? By default this parameter can be `null` to indicate that we want to use the existing `fallbackFontDict`, but if provided it allows the caller to override the default fallback font.

What do you think? /cc @Snuffleupagus to check if this is a good idea or not.

Reply:
> I'm not a big fan of this annotation detail leaking through to the generic evaluator.

Yeah, I completely agree with this sentiment; it just feels all-around weird to special-case the `PartialEvaluator` like that.

> Would it perhaps be better to let the caller (the annotation code) provide a `fallbackFont` dictionary parameter explicitly and pass that instead of this boolean?

That's a really nice idea, and one that I completely support!

Reply:
Is it really worth creating an object for such a corner case ? pr maybe we could then have a static fallback.

Reply:
Well, it's a more generic solution. The current solution is very specific to this single case, but puts annotation-specific details in the generic evaluator. Moving the fallback font dictionary itself to the annotation code, to which it belongs because it's specific to annotations, moves the edge case to there and ensures that the generic evaluator also gets a generic parameter, namely the fallback font dictionary to use. If it's not provided (by default), the existing fallback font dictionary is simply used.